src/migrations/03-create-requests.js

```diff
@@ -40,6 +43,10 @@ module.exports = {
         key: 'id'
       }
     },
+    return_trip: {
+      type: Sequelize.BOOLEAN,
+      allowNull: true,
```

Comment:
had a default value here which indicates that a request is not a return trip. Thus remove the allowNull attribute.

Reply:
done..thanks